.github/workflows/ci-sharktank.yml

```diff
@@ -149,7 +150,7 @@ jobs:
             sharktank/tests/models/vae/vae_test.py \
             sharktank/tests/models/llama/quark_parity_test.py \
             --durations=0 \
-            --timeout=800
+            --timeout=10000
```

Comment:
Nearly 3 hours? That's far too long for a job that runs on pull requests. Can this be set back to 800? Or 900 like the other job (15 minutes)